src/askem_beaker/contexts/model_configuration/procedures/python3/get_schema.py

```diff
@@ -0,0 +1,34 @@
+import requests
+import json
+import os
+
+# Fetch the Swagger spec
+url = f"{os.environ['HMI_SERVER_URL']}/v3/api-docs"
```

Comment:
If we want to future-proof, env variables should be collected in the context.py file passed in as a template render argument as we cannot guarantee that the procedure will run in the same environment as the beaker kernel.

Reply:
Good catch I'll address that

Reply:
Fixed that